auth-web/src/components/datatable/components/HeaderFilter.vue

```diff
@@ -86,7 +86,9 @@ export default defineComponent({
     }
 
     function applyFilters (props, state, header) {
-      props.updateFilter(header.col, header.customFilter.value)
+      if (props.updateFilter) {
```

Comment:
Probably don't need this now, since the you're overriding the slot and the default is an empty function